app/src/main/java/org/simple/clinic/contactpatient/ContactPatientBottomSheet.kt

```diff
@@ -240,6 +252,10 @@ class ContactPatientBottomSheet : BaseBottomSheet<
     removeAppointmentView.disableRemoveAppointmentDoneButton()
   }
 
+  private fun backPressed() {
+    hotEvents.onNext(BackClicked)
```

Comment:
We might be able to remove the need for this, I think it was originally added because of the activity and screen lifecycle interactions which should not be a problem anymore.

Reply:
For now, it's needed, because we are handling back clicks in `ContactPatientUpdate#backClicks` to either update UI or go back. Once we separate out the 3 UI modes into their own sheets/screens, we can then remove this.